posts/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+from . import views
+
+
+urlpatterns = [
+    path('', views.feed, name='Posts-feed'),
+    path('about/', views.about, name='Posts-about'),
+    path('search/', views.search, name='Posts-search'),
```

Comment:
Although path names aren't [restricted to valid Python names](https://docs.djangoproject.com/en/4.0/topics/http/urls/#naming-url-patterns-1), I still think it should be lowercase

Reply:
fixed